### DISCUSSION TITLE
A newScreen is always create when launching a phpunit test using phpstorm and ddev-docker

### DISCUSSION THREAD

**Discussion by fredattack** [spatie/ray#862] [Q&A]
## Environment
- PhpStorm Version: 2023.2.3
- DDEV-Docker Version: 1.22.4
- PHP Version: 8.1
- Laravel Version: 9.2
- Ray: 1.33 - Version 2.7.5 (2.7.5)
- Operating System: MacOs

## Description
When running PHPUnit tests in PhpStorm using ddev-docker, a new instance of `newScreen Ray` is being created each time. This behavior is unexpected and seems to occur regardless of the test being run. this trouble doesn't appear in other app env. 

Would appreciate any insights or solutions to prevent this unintended behavior. Thank you!

**Comment by freekmurze** [comment]
Take a look at the `setUp` method of your testsuite. Is there perhaps a stray ray call there?

**Reply by fredattack** [reply]
hi, thanks for the answer, but it 's a fresh testing env . I doesn't change anything.

**Reply by freekmurze** [reply]
There must be something there in your testsuite calling Ray. The desktop app doesn't do anything in regards to PHPUnit automatically.
